README.md

```diff
@@ -64,37 +62,136 @@ let webview = builder.build(&window).unwrap();
 let webview = builder.build_gtk(window.gtk_window()).unwrap();
 ```
 
-For more information, please read the crate [documentation](https://docs.rs/wry) .
+### Child webviews
+
+You can use [`WebView::new_as_child`] or [`WebViewBuilder::new_as_child`] to create the webview as a child inside another window. This is supported on
```

Comment:
Notice that all these intradoc links, while nice in Rust code, become dangling unfortunately :sweat_smile:.